### PR TITLE
kudeadm: Replace "master" with "control plane node"

### PR DESCRIPTION
**Pull request by devurandom**
This replaces the word "master" with "control plane node" in order to use
more inclusive language.

Also uses consistent wording / spelling in this context.

References: #21621
Suggested-by: Lubomir I. Ivanov <lubomirivanov@vmware.com>

### CODE REVIEW

**Comment by devurandom** [comment]
> To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **savitharaghunathan**
> You can assign the PR to them by writing `/assign @savitharaghunathan` in a comment when ready.

/assign @savitharaghunathan

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ LoadBalancer, or with dynamic PersistentVolumes.
 For both methods you need this infrastructure:
 
 - Three machines that meet [kubeadm's minimum requirements](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) for
-  the masters
+  the control plane nodes
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:39]: @devurandom `control-plane` is more widely used, so better have with it with the dash in this document too:
https://github.com/kubernetes/website/search?q=%22control-plane%22&unscoped_q=%22control-plane%22

**Review by neolit123 — COMMENTED** [review]
thanks for the PR. added one comment.
/sig cluster-lifecycle

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (devurandom) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:39]: I get 476 results for "control plane", which includes "control-plane": https://github.com/kubernetes/website/search?q=%22control+plane%22&unscoped_q=%22control+plane%22
And 205 results for "control-plane": https://github.com/kubernetes/website/search?q=%22control-plane%22&unscoped_q=%22control-plane%22

So it is actually quite even. I will undo the change from "control-plane" to "control plane" in this document.

**Comment by neolit123** [comment]
i think we have "control-plane" more consistent in the kubeadm related pages, but it's also in the source code of kubeadm for user facing text:
https://github.com/kubernetes/kubernetes/search?q=path%3A%2Fcmd%2Fkubeadm+%22control-plane%22&unscoped_q=path%3A%2Fcmd%2Fkubeadm+%22control-plane%22

so my preference is "control-plane" for this change.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ in the kubeadm config file.
     scp /etc/kubernetes/pki/apiserver-etcd-client.key "${CONTROL_PLANE}":
     ```
 
-    - Replace the value of `CONTROL_PLANE` with the `user@host` of the first control plane machine.
+    - Replace the value of `CONTROL_PLANE` with the `user@host` of the first control plane node.
```
Comment (devurandom) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:227]:
```suggestion
    - Replace the value of `CONTROL_PLANE` with the `user@host` of the first control-plane node.
```

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (devurandom) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:39]:
```suggestion
  the control-plane nodes
```

**Comment by devurandom** [comment]
> so my preference is "control-plane" for this change.

Done in 4a78a30f6d9220dd37d763525028509dbd08ba4b

**Comment by neolit123** [comment]
Thanks. Please squash the commits to one.

**Comment by savitharaghunathan** [comment]
Ping @neolit123 for tech review lgtm :)

**Review by neolit123 — COMMENTED** [review]
/lgtm
thanks

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**